Library/Homebrew/formula_installer.rb

```diff
@@ -608,6 +610,11 @@ def finish
     ohai "Summary" if verbose? || show_summary_heading?
     puts summary
 
+    # Updates the cache for a particular formula after doing an install
+    DatabaseCache.new(:linkage) do |database_cache|
+      LinkageChecker.new(keg, database_cache, false, formula)
```

Comment:
Will this update/populate the cache even if it's empty? Would be ideal if this is a no-op if the cache is empty.

Reply:
Yeah, currently this updates the cache even if it's empty. I'll update it so it's a no-op if the cache is empty.